packages/react/src/SubNav/SubNav.tsx

```diff
@@ -62,96 +130,182 @@ export type SubNavProps = {
+      <SubNavProvider>
+        <nav
+          ref={navRef}
+          id={id}
+          className={clsx(
+            styles.SubNav,
+            isOpenAtNarrow && styles['SubNav--open'],
+            hasShadow && styles['SubNav--has-shadow'],
+            fullWidth && styles['SubNav--full-width'],
+            className,
+          )}
+          data-testid={testId || testIds.root}
         >
-          {LinkChildren}
-          {ActionChild && <li className={styles['SubNav__action-container']}>{ActionChild}</li>}
-        </ul>
-      )}
-      <button
-        className={styles['SubNav__overlay-toggle']}
-        data-testid={testIds.button}
-        onClick={handleMenuToggle}
-        aria-expanded={isOpenAtNarrow ? 'true' : 'false'}
-        aria-controls={idForLinkContainer}
-        aria-label={`${isOpenAtNarrow ? 'close' : 'open'} navigation menu`}
-      >
-        {isOpenAtNarrow ? (
-          <XIcon className={styles['SubNav__overlay-toggle-icon']} size={24} />
-        ) : (
-          <div className={styles['SubNav__overlay-toggle-content']}>
-            <Text as="span">{activeLink && activeLink.props.children}</Text>
-            <ChevronDownIcon className={styles['SubNav__overlay-toggle-icon']} size={24} />
+          <div ref={rootRef} className={styles['SubNav--header-container-outer']}>
+            <div className={styles['SubNav__header-container']}>
+              {HeadingChild && <div className={styles['SubNav__heading-container']}>{HeadingChild}</div>}
+              <span role="separator" className={styles['SubNav__heading-separator']} aria-hidden>
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="8"
+                  height="16"
+                  viewBox="0 0 8 16"
+                  fill="none"
+                  aria-hidden
+                >
+                  <g clipPath="url(#clip0_50_1307)">
+                    <path d="M0 15.2992L5.472 0.701172H7.632L2.16 15.2992H0Z" fill="currentColor" />
+                  </g>
+                  <defs>
+                    <clipPath id="clip0_50_1307">
+                      <rect width="7.632" height="14.598" transform="translate(0 0.701172)" />
+                    </clipPath>
+                  </defs>
+                </svg>
+              </span>
+              {activeLink && activeLinklabel && !isLarge && (
+                <button
+                  className={clsx(
+                    styles['SubNav__overlay-toggle'],
+                    isOpenAtNarrow && styles['SubNav__overlay-toggle--open'],
+                  )}
+                  data-testid={testIds.button}
+                  onClick={isOpenAtNarrow ? closeMenuCallback : handleMenuToggle}
+                  aria-expanded={isOpenAtNarrow ? 'true' : 'false'}
+                  aria-controls={idForLinkContainer}
+                  aria-label={`${isOpenAtNarrow ? 'close' : 'open'} navigation menu`}
```

Comment:
I recall us receiving guidance from a11y design team on this being okay, that it should have an aria-label to provide better context on what users should do here rather than showing an arbitrary value like "Overview". Has that guidance changed?

Reply:
If we've had that guidance before then I'd say stick to that. It feels unintuitive to me because I can see something on the screen that I can't have the screen reader read out to me.

I'll reach out for guidance on this so I know for future reference, but don't let this block this PR